xbmc/TextureCache.cpp

```diff
@@ -100,7 +100,8 @@ std::string CTextureCache::GetCachedImage(const std::string &image, CTextureDeta
 
 bool CTextureCache::CanCacheImageURL(const CURL &url)
 {
-  return (url.GetUserName().empty() || url.GetUserName() == "music");
+  return url.GetUserName().empty() || url.GetUserName() == "music" ||
+          StringUtils::StartsWith(url.GetUserName(), "video_");
```

Comment:
typo on `video_`?

Reply:
no, it's image://video_[fanart|poster|thumb|etc]@file